spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/BigDLSerde.scala

```diff
@@ -223,6 +225,46 @@ object BigDLSerDe extends BigDLSerDeBase with Serializable {
     }
   }
 
+  private[python] class JActivityPickler extends BigDLBasePickler[JActivity] {
+    private def doConvertTable(table: Table): Any = {
+      val sortedList = table.getState().toList.sortWith(
```

Comment:
Instead of sorting, I think you should do something like `table.toSeq().toList`

Reply:
Within `toSeq` it just retrieve the key value starting with 1 same as sorting, but it also add restriction that the value should only be `Tensor[T]` instead of `Activity`. 
Should we update that method `toSeq` to return `Seq[Activity]`? if so, I think we also need to refactor some code to adapt this changes.
```
def toSeq[T]: Seq[Tensor[T]] = {
```

Reply:
let me update `toSeq` method to return Seq[Activity]

Reply:
You can either update `toSeq`, or retrieve each element using index 1 to N. Either way, we should avoid sorting.